### ISSUE TITLE
Daemon not starting, when special characters are in datadir

### ISSUE THREAD

**Issue by RyuMaster** [xaya/xaya#79]
Tested on Windows 10 64x; When I set up datadir with special characters in path, no files are being created in that folder at all, not even debug log, it just stays empty

**Comment by domob1812** [comment]
That is strange - the regression tests [explicitly include special characters](https://github.com/xaya/xaya/blob/c9360ff7707a5fdb414bdc7ed6d133fe62d19f01/test/functional/test_runner.py#L294).  At least on GNU/Linux, they work fine for me.  (But perhaps there was some upstream Bitcoin bug, as a consequence of which this was added to the tests.)

Have you ever tried running the regression tests (`test/functional/test_runner.py`) on Windows?  Can you please try what that does and if it fails all tests with the same error you are seeing?

**Comment by RyuMaster** [comment]
This one could be due to *bat file wallet uses, I will make more tests and get back

**Comment by RyuMaster** [comment]
Yes, this is *.bat file error. For some reason, it works only after I run it for the second time. As this is not related to XAYA core, issue can be closed.
